source/SIL.AppBuilder.Portal.Backend.Tests/Acceptance/EventDispatcher/EntityHandler/ProjectHookHandlerTest.cs

```diff
@@ -72,62 +72,56 @@ public void Notify_Project_Insert()
         {
             BuildTestData();
 
-            var handler = _fixture.GetService<IEntityHookHandler<Project>>();
-            var hubContext = _fixture.GetService<IHubContext<ScriptoriaHub>>();
-            var mockScriptoriaHub = Mock.Get(hubContext);
-            mockScriptoriaHub.Reset();
+            var handler = _fixture.GetService<IEntityHookHandler<Project, int>>();
```

Comment:
Not sure how helpful it would be, but this kind of pattern with the hubContext seems to be repeated a lot.  Maybe refactor this into some shared function?